images/linux/scripts/installers/terraform.sh

```diff
@@ -5,7 +5,7 @@
 ################################################################################
 
 # Install Terraform
-TERRAFORM_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/terraform | jq -r .current_version)
+TERRAFORM_VERSION=$(curl -s https://api.releases.hashicorp.com/v1/releases/terraform/latest | jq -r .version)
```

Comment:
Let's update the filter and get a link from json:
`'.builds[] | select((.arch=="amd64") and (.os=="linux")).url'`